_includes/payment-order-other-features.md

```diff
@@ -1357,6 +1358,20 @@ with the request parameter `instrument`.
 
 {% include settlement-reconciliation.md api_resource="paymentorders" %}
 
+## 3D Secure 2.0
+
+When dealing with card payments, 3D Secure authentication of the
+cardholder is an essential topic. 3D Secure 2.0 is an improved version of the
+old protocol, now allowing frictionless payments where transactions can be
+completed without input from the cardholder. Therefore, there are certain fields
+that should be included when implementing 3D Secure 2.0. These are listed below: 
+*   `payment.riskIndicator`
+*   `payment.cardholder`
+*   `paymentorder.payer.email` 
+*   `paymentorder.payer.msisdn`
+*   `paymentorder.payer.workPhoneNumber` 
+*   `paymentorder.payer.homePhoneNumber`
```

Comment:
Perhaps we should enlist all fields within `payer` by just providing a full JSON code example?